### ISSUE TITLE
Accessibility on Windows

### ISSUE THREAD

**Issue by chrisnorman7** [bambulab/BambuStudio#7733]
### Bambu Studio Version

I can't access the help menu to find out

### Where is the application from?

Bambu Lab Official website

### OS version

Microsoft Windows [Version 10.0.26100.4770]

### Additional system information

16-core AMC PC with 32 gb RAM

### Printer

Bambu Lab A1C

### How to reproduce

Hi,
I'm using Bambu Studio with NVDA, and many of the controls are unlabelled. For example, I can't see there is a help menu. I suspect it's somewhere near the top of the widget tree where NVDA says "Unknown".

Are there plans to improve the accessibility of the application?

### Actual results

Most of the interface is unalbelled. I cannot stop print jobs without using the Bambu Handy app on my phone.

### Expected results

I should be able to use at least the basicv parts of the application. I don't expect the GL canvas to be made accessible, but buttons should be labelled properly. Things like the colour selector.

### Project file & Debug log uploads

[file.zip](https://github.com/user-attachments/files/21557235/file.zip)

### Checklist of files to include

- [ ] Log file
- [ ] Project file

**Comment by Mujin-code** [comment]
Hello, thank you for your feedback. We have attempted to reproduce the issue based on the steps you described but were unable to do so. To help us investigate further,
1.Could you please provide the model project file along with a video showing the operation? You can use a simple screen recorder such as [ScreenToGif](https://github.com/NickeManarin/ScreenToGif/releases/tag/2.41.1).
2.Could you pleaseprovide your log file so that we can further analyze the issue?
The steps to obtain the logs are as follows：https://wiki.bambulab.com/en/x1/troubleshooting/bambu-studio-log
This will help us check whether it’s a rendering issue or something else.
Thanks a lot for your help and patience!
